nipype/interfaces/spm/base.py

```diff
@@ -178,17 +180,18 @@ def getinfo(klass, matlab_cmd=None, paths=None, use_mcr=None):
             returns None of path not found
         """
 
-        if klass._name and klass._path and klass._version:
+        use_mcr = use_mcr or 'FORCE_SPMMCR' in os.environ
+        matlab_cmd = matlab_cmd or ((use_mcr and os.getenv('SPMMCRCMD'))
+                      or os.getenv('MATLABCMD', 'matlab -nodesktop -nosplash'))
+
+        if klass._name and klass._path and klass._version and \
+                klass._command == matlab_cmd and klass._paths == paths:
```

Comment:
This is related to your discussion with @effigies. Shouldn't we add some warnings if `klass._command != matlab_cmd` or `klass._paths != paths`. Not sure how likely this might eb due to mistake.

Reply:
i'm not a big fan of warnings. almost no one reads it.

Reply:
well, you're probably right, but personally I sometimes read them if I feel hopeless...